packages/useraccounts-core/lib/methods.js

```diff
@@ -89,8 +88,14 @@ if (Meteor.isServer) {
                 }).data;
 
                 if (!apiResponse.success) {
-                    throw new Meteor.Error(403, AccountsTemplates.texts.errors.captchaVerification,
-                      apiResponse['error-codes'] ? apiResponse['error-codes'].join(", ") : "Unknown Error.");
+                    try {
+                        var msg = apiResponse['error-codes'].join(", ");
+                    } catch(e) {
+                        console.error('Exception Caught: ',e);
+                        var msg = "Unknown Error.";
+                    }
+
+                    throw new Meteor.Error(403, AccountsTemplates.texts.errors.captchaVerification, msg);
```

Comment:
In travis the .join triggers an undefined exception:
https://travis-ci.org/ecohealthalliance/mantle#L190

This only happens in travis, and not in a mac environment.